hack/update-crds.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+
+#
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Run operator-sdk generate k8s and operator-sdk generate openapi to update code after crd changes.
+
+if ! [ -x "$(command -v operator-sdk)" ]; then
+  echo 'Error: operator-sdk is not installed.' >&2
+  exit 1
+fi
+
+echo "run operator-sdk generate k8s"
```

Comment:
consider using "set -x" instead

Reply:
by "instead" i meant that there is no need for "echo"s

Reply:
can you explain why?

Reply:
since "-x" already echoes the commands

Reply:
ok, got you.